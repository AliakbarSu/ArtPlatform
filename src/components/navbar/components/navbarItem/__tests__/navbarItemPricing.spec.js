import { shallowMount, createLocalVue } from "@vue/test-utils";
import NavbarItemPricing from "../navbarItemPricing.vue";
import VueRouter from "vue-router";

const localVue = createLocalVue();
localVue.use(VueRouter);

describe("navbarItemPricing.vue", () => {
  let wrapper = null;
  beforeEach(() => {
    wrapper = shallowMount(NavbarItemPricing, {
      localVue
    });
  });

  it("should display 'Pricing'", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
