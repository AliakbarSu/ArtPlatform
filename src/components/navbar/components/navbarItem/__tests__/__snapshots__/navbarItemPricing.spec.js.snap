// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbarItemPricing.vue should display 'Pricing' 1`] = `
<div class="o-courses">
  <router-link-stub to="/pricing" tag="h2" event="click" class="o-courses__item">
    Pricing
  </router-link-stub>
</div>
`;
